justspacesapp/urls.py

```diff
@@ -13,9 +13,98 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
-from django.conf.urls import url, include
-from justspacesapp.views import CreateNewSurvey
+from django.conf.urls import url
+from django.utils.translation import ugettext_lazy as _
+from justspacesapp.views import AgencyCreate, LocationCreate, StudyCreate, \
+                                SurveyListEdit, SurveyListView
+import fobi.views
 
 urlpatterns = [
-    url(r'^$', CreateNewSurvey.as_view(), name='create-new-survey'),
+    url(r'^$',
+        SurveyListView.as_view(),
+        name='home'),
+
+    url(r'agencies/create$',
+        AgencyCreate.as_view(),
+        name='agencies-create'),
+
+    url(r'locations/create$',
+        LocationCreate.as_view(),
+        name='locations-create'),
+
+    url(r'studies/create$',
+        StudyCreate.as_view(),
+        name='studies-create'),
+
+    # Create new survey
+    url(r'surveys/create$',
+        view=fobi.views.create_form_entry,
+        name='fobi.create_form_entry'),
+
+    # Create survey element
```

Comment:
I think this URL should be restructured. You have a survey, and the survey has elements that belong to it, right? If so, you're URL should be:

`surveys/{survey id}/elements/{element id}/create`

Otherwise, you have two nondescript IDs side-by-side. 

The same goes for `surveys/handlers` below.

Reply:
The elements themselves have a survey ID attribute, so it's not in the url. This is `surveys/elements/create/{element} id}/{type of element}`. For example: `/surveys/elements/create/23/boolean/`. Would you still suggest changing this?

Reply:
Oh! I see. To be clear: one element can belong to multiple surveys? and one survey can have multiple elements? i.e., an element does not belong to a particular survey...

If so, then we do not need `surveys` at the front of the URL. The url should be: `/elements/create/{element} id}/{type of element}`

Reply:
An element belongs to just one survey, but surveys can have unlimited elements. Your example URL looks good and I'll switch it!